packages/core-modules/contracts/submodules/election/ElectionCredentials.sol

```diff
@@ -0,0 +1,89 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@synthetixio/core-contracts/contracts/proxy/UUPSProxy.sol";
+import "../../tokens/CouncilToken.sol";
+import "./ElectionBase.sol";
+
+contract ElectionCredentials is ElectionBase {
+    using SetUtil for SetUtil.AddressSet;
+
+    function _createCouncilToken(string memory tokenName, string memory tokenSymbol) internal {
+        CouncilToken implementation = new CouncilToken();
+
+        UUPSProxy proxy = new UUPSProxy(address(implementation));
+
+        CouncilToken token = CouncilToken(address(proxy));
+
+        token.nominateNewOwner(address(this));
+        token.acceptOwnership();
+
+        token.initialize(tokenName, tokenSymbol);
+
+        _electionStore().councilToken = address(token);
+    }
+
+    function _addCouncilMembers(SetUtil.AddressSet storage newMembers) internal {
+        uint numNewMembers = newMembers.length();
+
+        for (uint newMemberPosition = 1; newMemberPosition <= numNewMembers; newMemberPosition++) {
+            _addCouncilMember(newMembers.valueAt(newMemberPosition));
+        }
+    }
+
+    function _removeAllCouncilMembers() internal {
+        SetUtil.AddressSet storage members = _electionStore().councilMembers;
+
+        uint numMembers = members.length();
+
+        for (uint memberIndex = 0; memberIndex < numMembers; memberIndex++) {
+            _removeCouncilMember(members.valueAt(1));
```

Comment:
I think here's a typo:
```suggestion
            _removeCouncilMember(members.valueAt(memberIndex + 1));
```

Reply:
(Not sure if it should be memberIndex or memberIndex + 1)

Reply:
It works by always removing the first member until there are no more members.

Reply:
could you comment that in the code to easy reading it (and not think there's a typo)

Reply:
Done